docs/operations/single-server.md

```diff
@@ -42,37 +43,43 @@ The startup scripts for these example configurations run a single ZK instance al
 
 The example configurations run the Druid Coordinator and Overlord together in a single process using the optional configuration `druid.coordinator.asOverlord.enabled=true`, described in the [Coordinator configuration documentation](../configuration/index.md#coordinator-operation).
 
+The `start-druid` is a generic launch script for starting druid services on single server, it accepts optional arguments like services, memory and config.
+All reference configurations can be acheived by passing appropriate arguments to this script.
+Existing launch scripts are deprecated and will be removed in the next release. 
+
+
 While example configurations are provided for very large single machines, at higher scales we recommend running Druid in a [clustered deployment](../tutorials/cluster.md), for fault-tolerance and reduced resource contention.
 
 ## Single server reference configurations
 
-### Nano-Quickstart: 1 CPU, 4GiB RAM
+### Nano: 1 CPU, 4GiB RAM
 
-- Launch command: `bin/start-nano-quickstart`
```

Comment:
Please revert these changes -- this part of the doc is meant to describe the nano, micro, etc, bundled configs, which have their own run commands and config directories. We may change that at some point but we haven't yet.